src/Security/MemberAuthenticator/LostPasswordHandler.php

```diff
@@ -242,6 +242,8 @@ protected function sendEmail($member, $token)
             ))
             ->addData('PasswordResetLink', Security::getPasswordResetLink($member, $token))
             ->setTo($member->Email);
+
+        $this->extend('updateForgotPasswordEmail', $email);
```

Comment:
```suggestion
        $this->extend('updateForgotPasswordEmail', $email, $member);
```
This email is for a specific member, so we should pass the member reference to the extension.
Not needed for the other one since thats being extended from the member itself.